src/server/requirements.ts

```diff
@@ -66,14 +66,14 @@ function checkJavaRuntime(context: ExtensionContext): Promise<string> {
       return resolve(javaHome);
     }
     //No settings, let's try to detect as last resort.
-    findJavaHome({ allowJre: true }, function (err, home) {
-      if (err) {
-        openJDKDownload(reject, 'Java runtime could not be located.');
-      }
-      else {
-        resolve(home);
-      }
-    });
+    const javaRuntimes = await findRuntimes({ checkJavac: true, withVersion: true, withTags: true });
```

Comment:
I don't think that we need to check for `javac` here, since we just need a JRE (to run java software) not a full JDK (to run and develop/compile java software)